src/codeu/chat/client/commandline/Chat.java

```diff
@@ -361,12 +403,16 @@ public void invoke(Scanner args) {
     //
     panel.register("m-add", new Panel.Command() {
       @Override
-      public void invoke(Scanner args) {
-        final String message = args.hasNext() ? args.nextLine().trim() : "";
-        if (message.length() > 0) {
-          conversation.add(message);
-        } else {
+      public void invoke(List<String> args) {
+        if(args.size() != 1){
+          System.out.println("ERROR: Failed to add new user");
+        }
+        else{
+          String message = args.get(0);
+          if (message.length() < 0) {
```

Comment:
This if is not needed, we sure that args.get(0) will have something because we are using the tokenizer which ensures that this won't be empty.